modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -5,18 +5,19 @@
 module ClaimsApi
   module V2
     class VeteranIdentifierController < ClaimsApi::V2::ApplicationController
-      # TODO: REMOVE BEFORE IMPLEMENTATION
-      skip_before_action :authenticate, only: %i[find]
-      ICN_FOR_TEST_USER = '1012667145V762142'
-      # TODO: REMOVE BEFORE IMPLEMENTATION
-
       def find
         raise ::Common::Exceptions::Unauthorized if request.headers['Authorization'].blank?
 
         validate_request
```

Comment:
These validations are good enough for this endpoint but I wanted to investigate something like ActiveModel::Validations eventually, especially for our larger more complex endpoints.
Worth using this simpler endpoint as a proof of concept?
https://blog.kalina.tech/2019/12/nested-api-parameter-validation-in-rails.html

Reply:
Used the example link you provided to layout an initial pattern using `ActiveModel::Validations`.